services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/client/S3ClientWrapper.java

```diff
@@ -62,6 +62,10 @@ public boolean bucketExists(String bucketName) {
   }
 
   @Override
+  @Retryable(
```

Comment:
Do we use this `@Retryable` in upload/download as well? If not, why?

Reply:
We don't. And I don't know why. 🙂